.github/workflows/nodejs.yml

```diff
@@ -28,7 +27,7 @@ jobs:
       - run: npm ci
       - run: sudo docker run --name mongo -d -p 27017:27017 mongo
       - run: npm run build --if-present
-      - run: npm run format
```

Comment:
fromat was autoformatting the code when added to master, lint only checks if the code is formatted and fails if not right?

Reply:
lint does formatting as well since I configured the linter script to also fix things :)